app/views/state_file/questions/id_grocery_credit/edit.html.erb

```diff
@@ -0,0 +1,97 @@
+              <div class="tight-checkboxes spacing-above-0 spacing-below-0">
+                <%= f.cfa_checkbox(:primary_has_grocery_credit_ineligible_months, current_intake.primary.full_name, options: { checked_value: "yes", unchecked_value: "no", "data-follow-up": "#primary_months_ineligible_for_grocery_credit" }) %>
+                <% if current_intake.filing_status_mfj? %>
+                  <%= f.cfa_checkbox(:spouse_has_grocery_credit_ineligible_months, current_intake.spouse.full_name, options: { checked_value: "yes", unchecked_value: "no", "data-follow-up": "#spouse_months_ineligible_for_grocery_credit" }) %>
+                <% end %>
+                <%= f.fields_for :dependents do |ff| %>
+                  <% dependent = ff.object %>
+                  <%= ff.cfa_checkbox(:id_has_grocery_credit_ineligible_months, dependent.full_name, options: { checked_value: "yes", unchecked_value: "no", "data-follow-up": "#dependent_#{dependent.id}_months_ineligible_for_grocery_credit" }) %>
+                <% end %>
+              </div>
+            </div>
+          </div>
+          <div class="question-with-follow-up__follow-up" id="primary_months_ineligible_for_grocery_credit">
+            <div class="white-group">
+              <p><%= t(".need_to_know_you_html") %></p>
+              <p><%= t(".example_months") %></p>
+              <%= f.cfa_select(:primary_months_ineligible_for_grocery_credit, t(".total_number_months"), (1..12).to_a, label_class: 'form-label--bold', include_blank: t('general.select_prompt')) %>
+            </div>
+          </div>
+          <% if current_intake.filing_status_mfj? %>
+            <div class="question-with-follow-up__follow-up" id="spouse_months_ineligible_for_grocery_credit">
+              <div class="white-group">
+                <p><%= t(".need_to_know_spouse_html") %></p>
+                <p><%= t(".example_months") %></p>
+                <%= f.cfa_select(:spouse_months_ineligible_for_grocery_credit, t(".total_number_months"), (1..12).to_a, include_blank: t('general.select_prompt')) %>
+              </div>
+            </div>
+          <% end %>
+          <%= f.fields_for :dependents do |ff| %>
+            <% dependent = ff.object %>
+            <div class="question-with-follow-up__follow-up" id="dependent_<%= dependent.id %>_months_ineligible_for_grocery_credit">
+              <div class="white-group">
+                <p><%= t(".need_to_know_dependent_html") %></p>
+                <p><%= t(".example_months") %></p>
+                <%= ff.cfa_select(:id_months_ineligible_for_grocery_credit, t(".total_number_months"), (1..12).to_a, include_blank: t('general.select_prompt')) %>
+              </div>
+            </div>
+          <% end %>
+        </div>
+      </div>
+    </div>
+
+    <div class="reveal">
+      <p><a href="#" class="reveal__link"><%= t('.why_are_you_asking_heading') %></a></p>
```

Comment:
change to `<button class="reveal__button">` after merging main

Reply:
done